test_data/mysql_structured_report.json

```diff
@@ -1,151 +1,172 @@
 {
-   "summary":{
-      "text":"Schema conversion: POOR (65% of 17006 columns mapped cleanly) + some missing primary keys.\nData conversion: POOR (66% of 6000 rows written to Spanner).\n",
-      "rating":"POOR",
+   "summary": {
+      "text": "Schema conversion: POOR (29% of 17006 columns mapped cleanly) + some missing primary keys.\nData conversion: POOR (66% of 6000 rows written to Spanner).\n",
```

Comment:
What changed in the complexity logic? Why did 65% change to 29%?

Reply:
Previously missing primary key was not included in the calculation of total number of warnings but now they are included and this impacted the rating of the schema